src/gui/editors/AutomationEditor.cpp

```diff
@@ -273,23 +272,27 @@ void AutomationEditor::keyPressEvent(QKeyEvent * ke )
 			break;
 
 		case Qt::Key_Left:
-			if( ( m_timeLine->pos() -= 16 ) < 0 )
+			if(m_timeLine->timeline()->getTicks() - 16 > 0)
 			{
-				m_timeLine->pos().setTicks( 0 );
+				m_timeLine->timeline()->setTicks(m_timeLine->timeline()->getTicks() - 16);
 			}
-			m_timeLine->updatePosition();
+			else
+			{
+				m_timeLine->timeline()->setTicks(0);
+			}
+			m_timeLine->timeline()->updatePosition();
```

Comment:
It looks like `setTicks` already does the job of this method, so this would be emitting the same signal twice.